contracts/axelar-gateway/src/lib.rs

```diff
@@ -1,11 +1,12 @@
 #![no_std]
 
 // Allows using std (and its macros) in test modules
-#[cfg(test)]
+#[cfg(any(test, feature = "testutils"))]
 #[macro_use]
 extern crate std;
 
 pub mod error;
+pub mod event;
```

Comment:
Move this back, and add a self dependency on axelar-gateway in Cargo.toml similar to ITS here:
https://github.com/axelarnetwork/axelar-cgp-soroban/blob/main/contracts/interchain-token-service/Cargo.toml#L30

To enable testutils feature in integration tests to allow Event testutils to be picked up. By default Rust won't enable `cfg(test)` isn't available in Rust integration tests as opposed to unit tests.